build.gradle

```diff
@@ -4,15 +4,20 @@ import java.time.Instant
 import java.time.LocalDate
 
 plugins {
-	id 'java'
-	id 'org.springframework.boot' version '2.2.1.RELEASE'
-	id 'io.spring.dependency-management' version '1.0.8.RELEASE'
-	id 'com.github.ben-manes.versions' version '0.27.0'
+    id 'java'
+    id 'org.springframework.boot' version '2.2.1.RELEASE'
+    id 'io.spring.dependency-management' version '1.0.8.RELEASE'
+    id 'com.github.ben-manes.versions' version '0.27.0'
```

Comment:
missing `id 'org.owasp.dependencycheck' version '5.2.4'` too so that we can run `./gradlew dependencyCheckAnalyze` for owasp stuff